tests/conftest.py

```diff
@@ -0,0 +1,149 @@
+import configparser
+import os
+from datetime import datetime, timedelta
+
+import pytest
+from fastapi.testclient import TestClient
+from sqlalchemy import create_engine, text
+from sqlalchemy.orm import sessionmaker, Session
+from ml_warehouse.schema import (
+    Base,
+    PacBioRun,
+    PacBioRunWellMetrics,
+    PacBioProductMetrics,
+    Study,
+    Sample,
+)
+from lang_qc.main import app
+from lang_qc.db.mlwh_connection import get_mlwh_db
+
+test_ini = os.path.join(os.path.dirname(__file__), "testdb.ini")
+
+
+@pytest.fixture(scope="session")
+def config() -> configparser.ConfigParser:
+    # Database credentials for the test MySQL instance are stored here. This
+    # should be an instance in a container, discarded after each test run.
+    test_config = configparser.ConfigParser()
+    test_config.read(test_ini)
+    yield test_config
```

Comment:
Is `yield` appropriate? Seems like you don't benefit from returning into this function. Can this not just be a `return`? Ref: https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#yield-fixtures-recommended